omc3/harpy/clean.py

```diff
@@ -31,9 +30,9 @@ def clean(harpy_input, bpm_data, model):
     Returns:
         Clean BPM matrix, its decomposition, bad BPMs summary and estimated BPM resolutions
     """
-    if not harpy_input.clean:
-        return bpm_data, None, [], None
     bpm_data, bpms_not_in_model = _get_only_model_bpms(bpm_data, model)
+    if not harpy_input.clean:
+        return bpm_data, None, bpms_not_in_model, None
     if bpm_data.empty:
```

Comment:
shouldn't this be also moved to the _get_only_model_bpms() i.e before the if return part?